pkg/reports/reports.go

```diff
@@ -0,0 +1,191 @@
+package reports
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"github.com/google/uuid"
+	"github.com/quay/clair-load-test/attacker"
+	"github.com/quay/clair-load-test/pkg/commons"
+	"github.com/quay/clair-load-test/pkg/manifests"
+	"github.com/quay/clair-load-test/pkg/token"
+	"github.com/quay/zlog"
+	"github.com/urfave/cli/v2"
+	"os"
+	"strconv"
+	"strings"
+)
+
+// Command line to handle reports functionality.
+var ReportsCmd = &cli.Command{
+	Name:        "report",
+	Description: "request reports for named containers",
+	Usage:       "clair-load-test report",
+	Action:      reportAction,
+	Flags: []cli.Flag{
+		&cli.StringFlag{
+			Name:    "host",
+			Usage:   "--host localhost:6060/",
+			Value:   "http://localhost:6060/",
+			EnvVars: []string{"CLAIR_TEST_HOST"},
+		},
+		&cli.StringFlag{
+			Name:    "uuid",
+			Usage:   "--uuid f519d9b2-aa62-44ab-9ce8-4156b712f6d2",
+			Value:   uuid.New().String(),
```

Comment:
In some cases the user might want to override the generated uuid, to help do a/b comparisons. ie `joes-fips-run-rhel9-aws`

Reply:
That's not a uuid -- is this just some ID for the run?

Reply:
Yes. If a user specifies it, it some ID to run. But if not specified, it will be randomly generated unique ID. We are providing this option, so that the user can easily identify the results of their run using this ID either in local or ES.